src/components/tile/tile.spec.js

```diff
@@ -102,6 +102,119 @@ describe("Tile", () => {
       });
     });
 
+    describe("custom borders", () => {
+      it('renders with expected border when borderVariant is set to "selected"', () => {
```

Comment:
This could be an `it.each` yo'll need to add a mechanism to resolve the correct styling for the given `borderVariant` the so I'm okay with you leaving as is if you prefer it this way

Reply:
Updated as per feedback